pcdsdevices/epics_motor.py

```diff
@@ -1579,6 +1579,22 @@ def __init__(self, prefix, *, ioc_base, **kwargs):
         super().__init__(prefix, **kwargs)
 
 
+class PI_M824(PVPositionerComparator):
+    """
+    Axis subclass for PI_M824_Hexapod class.
```

Comment:
If there's anything that makes sense to improve in this PR it's the docstring here with more information about what this is for. The referenced class here doesn't exist in pcdsdevices.